.github/workflows/install.yaml

```diff
@@ -14,14 +14,14 @@ jobs:
               with:
                   environment-file: conda.yaml
                   activate-environment: chemnlp
-                  python-version: 3.7
+                  python-version: 3.8
                   auto-update-conda: true
                   auto-activate-base: false
             - name: Validate yaml
               shell: bash -l {0}
               run: |
                   conda activate chemnlp
-                  python -m dev.validate data
+                  python -m src.chemnlp.data_val.validate data
             - name: Lint
```

Comment:
Perhaps it would be cleaner if we also simply run pre-commit here. I can also take care of that

Reply:
Sure, though this will make dealing with failures that `pre-commit` can't resolve more awkward? Also, you'd need to watch out for pushing, `pre-commit` running remotely and then pushing again without having integrating these changes. Not quite sure if there are failures modes to this workflow?